debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -1210,6 +1212,7 @@ protected CommonConnectorConfig(Configuration config, int defaultSnapshotFetchSi
         this.signalEnabledChannels = getSignalEnabledChannels(config);
         this.skippedOperations = determineSkippedOperations(config);
         this.taskId = config.getString(TASK_ID);
+        this.maxTasks = config.getInteger(TASKS_MAX, 1);
```

Comment:
Is this really guranteed that Kafka Connect will always propagate the connector infra level config to the connector configuration?